app/src/main/java/ly/loud/loudly/util/database/DatabaseUtils.java

```diff
@@ -102,11 +102,23 @@ static void deleteLocation(long id) throws DatabaseException {
         }
     }
 
-    static <T extends Attachment & MultipleNetwork> long saveAttachment(@NonNull T attachment, long postId) throws DatabaseException {
+    @NonNull
+    private static Link[] instancesToLinks(MultipleNetworkElement element) {
+        Link[] links = new Link[Networks.NETWORK_COUNT];
+        @SuppressWarnings("unchecked") // Java doesn't infer that it's an array of SingleNetworkElement
```

Comment:
Markdown dropped brackets. There were ArrayList ((T extends SingleNetworkElement)) and ArrayList ((Object))